src/test/java/io/openliberty/tools/intellij/lsp4jakarta/it/jaxrs/ResourceMethodTest.java

```diff
@@ -85,49 +85,25 @@ public void multipleEntityParamsMethod() throws Exception {
-                "\n    }\n}\n";
-        TextEdit te1 = JakartaForJavaAssert.te(0, 0, 25, 0, newText1);
-        CodeAction ca1 = JakartaForJavaAssert.ca(uri, "Remove all entity parameters except entityParam1", d, te1);
-
-        String newText2 = "/*******************************************************************************\n" +
-                "* Copyright (c) 2021 IBM Corporation.\n*\n" +
-                "* This program and the accompanying materials are made available under the\n" +
-                "* terms of the Eclipse Public License v. 2.0 which is available at\n" +
-                "* http://www.eclipse.org/legal/epl-2.0.\n*\n" +
-                "* SPDX-License-Identifier: EPL-2.0\n*\n* Contributors:\n" +
-                "*     Bera Sogut\n" +
-                "*******************************************************************************/\n\n" +
-                "package io.openliberty.sample.jakarta.jax_rs;\n\n" +
-                "import jakarta.ws.rs.DELETE;\n" +
-                "import jakarta.ws.rs.FormParam;\n\n" +
-                "public class MultipleEntityParamsResourceMethod {\n\n	" +
-                "@DELETE\n	" +
-                "public void resourceMethodWithTwoEntityParams(@FormParam(value = \"\") String nonEntityParam, int entityParam2) {" +
-                "\n        \n    }\n}\n";
-        TextEdit te2 = JakartaForJavaAssert.te(0, 0, 25, 0, newText2);
-        CodeAction ca2 = JakartaForJavaAssert.ca(uri, "Remove all entity parameters except entityParam2", d, te2);
+        JakartaJavaCodeActionParams codeActionParams = JakartaForJavaAssert.createCodeActionParams(uri, d1);
+        String newText1 = "/*******************************************************************************\n*" +
+                " Copyright (c) 2021 IBM Corporation.\n*\n*" +
+                " This program and the accompanying materials are made available under the\n* terms of the Eclipse Public License v. 2.0 which is available at\n* http://www.eclipse.org/legal/epl-2.0.\n*\n* SPDX-License-Identifier: EPL-2.0\n*\n* Contributors:\n*     Bera Sogut\n*******************************************************************************/\n\npackage io.openliberty.sample.jakarta.jax_rs;\n\nimport jakarta.ws.rs.BeanParam;\nimport jakarta.ws.rs.DELETE;\nimport jakarta.ws.rs.FormParam;\n\npublic class MultipleEntityParamsResourceMethod {\n\n\t@DELETE\n\tpublic void resourceMethodWithTwoEntityParams(String entityParam1, @FormParam(value = \"\") String nonEntityParam) {\n        \n    }\n\n\t@DELETE\n\tpublic void resourceMethodWithTwoBeanParams(@BeanParam RequestBean requestBean, @BeanParam SessionBean sessionBean) {\n        \n    }\n\n\t@DELETE\n\tpublic void resourceMethodWithBeanParamsAndEnityParams(String entityString, int entityInt, @BeanParam RequestBean requestBean, @BeanParam SessionBean sessionBean) {\n        \n    }\n}\n";
```

Comment:
Could you please reformat this line into separate lines like in the file you're replacing? This will make it easier to verify that the contents are correct.

Reply:
Looks good, thank you.